structure/build.sbt

```diff
@@ -0,0 +1,49 @@
+scalaVersion := "2.13.6"
+
+version := "1.0"
+
+libraryDependencies ++= Dependencies.All
+
+lazy val root = project
+  .enablePlugins(JUnitXmlReportPlugin)
```

Comment:
this plugin is enabled by default in SBT

Reply:
this was just to have example in sbt, but I better to remove it as this is not related to lecture much